lib/screens/home_page.dart

```diff
@@ -25,7 +25,7 @@ class _HomePageState extends State<HomePage> {
     return MaterialApp(
       home: Scaffold(
         appBar: AppBar(
-          title: const Text('Toto'),
+          title: const Text('Vivatech'),
```

Comment:
Possible to add official logo of Vivatech ?

Reply:
Not possible in the _**title**_ parameter of the MaterialApp widget